device/nokia/arm64-nokia_ixs7215_52xb-r0/pmon_daemon_control.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip_pcied": true
```

Comment:
@Pavan-Nokia why skip? No pcie devices?

Reply:
Yes that is right, We do not have any pcie devices on this platform (Nokia-7215-A1)